its/tests/src/test/java/its/SonarCloudTest.java

```diff
@@ -166,42 +167,41 @@ public static void prepare() throws Exception {
 
   @AfterClass
   public static void cleanup() {
-    adminWsClient.projects().bulkDelete(org.sonarqube.ws.client.project.SearchWsRequest.builder()
-      .setQuery("-" + randomPositiveInt)
-      .setOrganization(SONARCLOUD_ORGANIZATION)
-      .build());
+    adminWsClient.projects().bulkDelete(new org.sonarqube.ws.client.projects.BulkDeleteRequest()
+      .setQ("-" + randomPositiveInt)
+      .setOrganization(SONARCLOUD_ORGANIZATION));
   }
 
   private static void associateProjectToQualityProfile(String projectKey, String language, String profileName) {
-    adminWsClient.qualityProfiles().addProject(AddProjectRequest.builder()
-      .setProjectKey(projectKey(projectKey))
+    adminWsClient.qualityprofiles().addProject(new AddProjectRequest()
+      .setProject(projectKey(projectKey))
       .setLanguage(language)
       .setQualityProfile(profileName)
-      .setOrganization(SONARCLOUD_ORGANIZATION)
-      .build());
+      .setOrganization(SONARCLOUD_ORGANIZATION));
   }
 
   private static void restoreProfile(String profile) {
-    adminWsClient.qualityProfiles().restoreProfile(RestoreWsRequest.builder()
-      .setBackup(Paths.get("src/test/resources/" + profile).toFile())
-      .setOrganization(SONARCLOUD_ORGANIZATION)
-      .build());
+    File backupFile = new File("src/test/resources/" + profile);
+    // XXX can't use RestoreRequest because of a bug
```

Comment:
Any link to this bug? Like ticket number?

Reply:
Unfortunately, no. This issue is a regression introduced after a change in sonar-ws. I talked to someone from the SQ team about it, and they are not willing to fix it because this endpoint will probably be dropped soon. They now discourage the use of the restore feature, and they instead recommend to create a new quality profile and activate rules in it to achieve the same result.

I would like to find some time during this sprint to do the recommended move and get rid of this comment (and I agree it is really inaccurate)